Tiltfile

```diff
@@ -7,22 +7,13 @@ docker_build(
 )
 
 docker_build(
-  'local:log-service',
+  'local:logservice',
```

Comment:
For the record, I'd prefer to use `log-service` but we are using `logservice` in so many places that I decided to just stick with that instead of having to update all the k8s and proto code. It just seemed simpler to make this be the exception and at least use `logservice` consistently.